components/typography/metadata/typography-international.yml

```diff
@@ -1,12 +1,16 @@
 name: Typography (Internationalized)
+fastLoad: false
```

Comment:
This makes it so the right Typekit loads on the dev site when you click this example in the sidenav.